lib/tasks/data_maintenance.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
does this need to be in here?

Reply:
yep: https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-strinliteralgliteral-literal-true-do